src/main/java/calculator/domain/Calculator.java

```diff
@@ -0,0 +1,78 @@
+package calculator.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Calculator {
+    private Operands operands;
+    private Operators operators;
+
+    Calculator(Operands operands, Operators operators) {
+        this.operands = operands;
+        this.operators = operators;
+    }
+
+    public static Calculator create(String input) {
+        if (Objects.isNull(input) || input.isBlank()) {
+            throw new IllegalArgumentException("입력 값이 null이거나 빈 공백 문자일수 없습니다.");
+        }
+        return create(input.split(" "));
+    }
+
+    public static Calculator create(String[] operatorsOperands) {
```

Comment:
```suggestion
    private static Calculator create(String[] operatorsOperands) {
```
불필요한 접근을 막아줄 수 있겠네요 :)